doc/contribute.rst

```diff
@@ -18,6 +18,7 @@ source code is considered the 'installation'.
     git clone git@github.com:ComPWA/tensorwaves.git
     code tensorwaves
     conda env create
```

Comment:
actually dont you have to specify some name of the venv here?

Reply:
No, the name is specified in the `environment.yml` file. And you also don't have to use the flag `--file` because `environment.yml` is the default for that flag when running `conda env create`.

Reply:
ah i see